### ISSUE TITLE
Kicking All Players

### ISSUE THREAD

**Issue by StealthWolfz** [MeGysssTaa/ReflexIssueTracker#168]
**Issue type (console error/false positive/bypass/other):**
Other

**Do /reflex debug, and paste here the link you've got. Keep empty if on test server:**
https://p.reflex.rip/UBcY

**A video proving the bypass/false positive if reporting a bypass/false positive, a link to your server log if reporting a console error, or a link to your Reflex log if reporting a bypass/false positive:**
I'm not sure what to put here.

**Additional information (i.e. how to reproduce the issue you are experiencing etc.):**
Basically what happens is, Reflex will kick all players online at once. It seems to happen when the server tps drops lower than 17 and goes back up higher than 17 again although I'm not entirely sure though I think it's lag related.

**Comment by MeGysssTaa** [comment]
Video proofs, Reflex logs, server logs (with **debug** option enabled both in **server.properties** and in **spigot.yml**), kick messages (kick screen screenshots)?
